src/server/apis/dash-data.ts

```diff
@@ -23,7 +23,7 @@ export function getRouter(): express.Router {
  *  - ?cursor - page cursor
  */
 async function outgoingHandler(req: express.Request, res: express.Response) {
-  const loginDetails = await userModel.getLoginFromRequest(req);
+  const loginDetails = await userModel.getUserRecordFromRequest(req);
```

Comment:
Can we update these variable names too or update them as we change other things?